mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Bark.MixProject do
   def project do
     [
       app: :bark,
-      version: "1.1.2",
+      version: "1.2.0",
       elixir: "~> 1.9",
```

Comment:
This should probably be cahnged to 1.11 since warning was added then.  [Reference](https://github.com/smartrent/bark/settings)